### ISSUE TITLE
`gr.Markdown` does not support adding a container around it

### ISSUE THREAD

**Issue by irgolic** [gradio-app/gradio#8918]
- [x] I have searched to see if a similar issue already exists.

**Is your feature request related to a problem? Please describe.**  
It feels weird that `Textbox` and `JSON` add a container around them by default, but `Markdown` does not. There does not seem to be a clear way to add a container around `Markdown`. The same may apply to `HTML` as well, though it stands out as weird particularly in `Markdown`, that feels like it should be a stand-in replacement for `Textbox`.

**Describe the solution you'd like**  
Add a container around `gr.Markdown` by default to be consistent, or at least (for backwards compatibility sake) add a `container` kwarg that's `False` by default.

Alternatively, introduce a `gr.Container` so that we can manually add them. This is what Qt does (they call them Box) and it works intuitively.

**Comment by abidlabs** [comment]
Agreed this would be useful, I thought we already had an issue for this, but it seems not.

**Comment by irgolic** [comment]
Any progress? This is a blocker for me now

**Comment by abidlabs** [comment]
Not yet, but in the meantime, you could use HTML and CSS (which is supported in gr.Markdown) to add a custom border, kind of like I do in this Space: https://huggingface.co/spaces/abidlabs/llm-explorer
